### DISCUSSION TITLE
Audi smart phone interface issue

### DISCUSSION THREAD

**Discussion by solmaggiore** [Mr-MIBonk/M.I.B._More-Incredible-Bash#355] [Q&A]
Hello everyone,
I installed the smartphone interface option and loaded the Congo activator. Everything works, but when the parking sensors are activated, the application closes and the MMI window returns to the main menu.
MMi sw version is MHS2_EU_AU_K0137.
Thanks for your help.
dario

**Comment by suplstfdo** [comment]
Hi, sorry I have no answer to your issue. But have you succeeded in using M.I.B. on MHS2 Firmware?

I also have the MHS2 Firmware and can't find any hints wether M.I.B. works on it or not. Would really appreaciate some insights what you did and what worked.

Best Regards,
Michael
